projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/unlink-cases/unlink-cases.component.spec.ts

```diff
@@ -0,0 +1,193 @@
+import { RouterTestingModule } from '@angular/router/testing';
+import { of } from 'rxjs';
+import { PipesModule } from '../../../../../pipes';
+import { CaseEditComponent, CasesService } from '../../../../case-editor';
+import { CaseLink } from '../../domain/linked-cases.model';
+import { LinkedCasesService } from '../../services';
+import { UnLinkCasesComponent } from './unlink-cases.component';
+import createSpyObj = jasmine.createSpyObj;
+
+describe('UnLinkCasesComponent', () => {
+  let component: UnLinkCasesComponent;
+  let fixture: ComponentFixture<UnLinkCasesComponent>;
+  let casesService: any;
+  let linkedCasesService: any;
+  let nativeElement: any;
+  let caseEditComponentStub: any;
+
+  const caseInfo = {
+    case_id: '1682374819203471',
+    case_type: {
+      name: 'SSCS type',
+      jurisdiction: { name: '' }
+    },
+    state: { name: 'With FTA' },
+    tabs: [
+      {
+        id: 'linked_cases_sscs',
+        fields: [
+          {
+            value: [
+              {
+                caseReference: '1652112127295261',
+                modified_date_time: '2022-05-10',
+                caseType: 'Benefit_SCSS',
+                reasons: [
+                  {
+                    reasonCode: 'bail',
+                    OtherDescription: ''
+                  }
+                ]
+              },
+              {
+                caseReference: '1652111610080172',
+                modified_date_time: '2022-05-10',
+                caseType: 'Benefit_SCSS',
+                reasons: [
+                  {
+                    reasonCode: 'consolidated',
+                    OtherDescription: ''
+                  }
+                ]
+              },
+              {
+                caseReference: '1652111179220086',
+                modified_date_time: '2022-05-10',
+                caseType: 'Benefit_SCSS',
+                reasons: [
+                  {
+                    reasonCode: 'progressed',
+                    OtherDescription: ''
+                  },
+                  {
+                    reasonCode: 'familial',
+                    OtherDescription: ''
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  };
+
+  const linkedCases: CaseLink[] = [
+    {
+      caseReference: '1682374819203471',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1'
+    },
+    {
+      caseReference: '1682897456391875',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1'
+    }
+  ];
+
+  linkedCasesService = {
```

Comment:
@johnbenjamin-hmcts - could be declared immediately at Line 17

Reply:
@Daniel-Lam I have updated the code now.